incubator/gitea/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+description: 'Gitea: Go Git Service'
+name: gitea
+version: 1.3.2
+appVersion: 0.0.1
```

Comment:
It looks like you have the `version` and `appVersion` switched. The `version` is the version of the chart or, rather, the configuration to deploy to Kubernetes. The `appVersion` is the version of the underlying application being deployed.